packages/services/usage/src/usage.ts

```diff
@@ -212,21 +213,11 @@ export function createUsage(config: {
           rawOperationWrites.inc(numOfOperations);
           logger.info(`Flushed (id=%s, operations=%s)`, batchId, numOfOperations);
         }
-
-        changeStatus(Status.Ready);
       } catch (error: any) {
         rawOperationFailures.inc(numOfOperations);
 
         changeStatus(Status.Unhealthy);
-        logger.error(`Failed to flush (id=%s, error=%s)`, batchId, error.message);
-        Sentry.setTags({
-          batchId,
-          message: error.message,
-          numOfOperations,
-        });
-        Sentry.captureException(error);
```

Comment:
Moved this to the fallback queue since this is being handled.
I dont believe we should send handled errors to sentry.